auth.c

```diff
@@ -121,8 +121,9 @@ gboolean janus_auth_check_signature(const char *token, const char *realm) {
 }
 
 gboolean janus_auth_check_signature_contains(const char *token, const char *realm, const char *desc) {
-	if (!auth_enabled || auth_secret == NULL)
-		return FALSE;
+	if (!auth_enabled || auth_secret == NULL) {
+		return TRUE;
```

Comment:
Good catch, as if this is disabled returning `FALSE` might trigger unwanted errors.

Reply:
thank you